samples/docker-domain/README.md

```diff
@@ -12,6 +12,11 @@ This sample includes a basic WDT model, `simple-topology.yaml`, that describes t
 
 Another option is to use the WDT `discoverDomain` tool to create a model. This process is also described in the WDT project's README file. A user can use the tool to analyze an existing domain, and create a model based on its configuration. The user may choose to customize the model before using it to create a new Docker image.
 
+The sample model file includes variable tokens that are replaced with values from the provided 'simple-topology.properties'. The properties files can be created and modified using a text editor. The property file must contain the values that the Dockerfile sets as ENV variables to expose ports
+ and run a server in a docker container.
+ 
+The admin server credentials in the sample model contain file tokens that reference special property files. A special property file must contain only one property value pair. The sample includes the credentials secrets property files in the properties/docker_build directory. The files can be created and modified using a text editor; one file must contain the admin password and one file must contain the admin user. See the README file for more information on using property and file tokens in the WDT model.
+
```

Comment:
Note: Oracle recommends that the adminpass.properties, adminuser.properties and security.properties files be deleted or secured after the container and the WebLogic Server are started so that the user name and password are not inadvertently exposed.